packages/flutter/lib/src/foundation/binding.dart

```diff
@@ -83,6 +83,20 @@ abstract class BindingBase {
   /// [BindingBase], such as [TestWidgetsFlutterBinding], can override this
   /// accessor to return a different [ui.SingletonFlutterWindow] implementation,
   /// such as a [TestWindow].
+  ///
+  /// The `window` is a singleton meant for use by applications that only have a
+  /// single main window. In addition to the properties of [ui.FlutterWindow],
+  /// `window` provides access to platform-specific properties and callbacks
+  /// available on the [platformDispatcher].
+  ///
+  /// For applications designed for more than one main window, prefer using the
+  /// [platformDispatcher] to access available views via
+  /// [ui.PlatformDispatcher.views].
+  ///
+  /// Implementation note: multiple window support is not yet implemented, so
```

Comment:
nit: per style guide, let's remove the "empty prose" of "implementation note".

Reply:
OK, replaced it with "However, ".